docs/tutorial/rust/src/ideas_for_the_reader.md

```diff
@@ -3,7 +3,7 @@
 The game is visually a little bare. Here are some ideas how you could make further changes to enhance it:
 
 -   The tiles could have rounded corners, to look a little less sharp. The [border-radius](https://slint.dev/docs/slint/src/builtins/elements.html#rectangle)
-    property of _Rectangle_ can be used to achieve that.
+    property of _[Rectangle](https://slint.dev/releases/1.0.2/docs/slint/src/builtins/elements#rectangle)_ can be used to achieve that.
```

Comment:
This will redirect to the latest version

```suggestion
    property of _[Rectangle](https://slint.dev/docs/slint/src/builtins/elements#rectangle)_ can be used to achieve that.
```

Reply:
Whoops, sorry. Done and rebased.